sdk/cosmos/azure-cosmos/samples/examples_async.py

```diff
@@ -176,6 +176,39 @@ async def examples_async():
             print(json.dumps(item, indent=True))
         # [END delete_all_items_by_partition_key]
 
+            # insert items in a subpartitioned container
+            # [START upsert_items]
+            for i in range(1, 10):
+                await container.upsert_item(
+                    dict(id="item{}".format(i), state="WA", city="Redmond", zipcode=98052, population="1000000")
+                )
+            # [END upsert_items]
+
+            # Modify an existing item in the container
+            # [START update_item]
+            item = await container.read_item("item2", partition_key=["WA", "Redmond", 98052])
+            item["value"] = "2000000"
+            updated_item = await container.upsert_item(item)
+            # [END update_item]
+
+            # Query the items in a container using SQL-like syntax. This example
+            # gets all items whose product model hasn't been discontinued.
+            # [START query_items]
+            import json
+
+            async for item in container.query_items(
+                    query='SELECT * FROM location l WHERE l.state = "WA" AND l.population = "1000000"'
+            ):
+                print(json.dumps(item, indent=True))
+            # [END query_items]
+
+            # [START delete_items]
+            async for item in container.query_items(
+                    query='SELECT * FROM location l WHERE l.population = "2000000"'
+            ):
+                await container.delete_item(item, partition_key=["WA", "Redmond", 98052])
+            # [END delete_items]
```

Comment:
let's have this one match the example you put in the sync version - a key part of showing how subpartitioning works is showing how to create the subpartitioned container